install/kubernetes/cilium/values.yaml

```diff
@@ -1332,6 +1332,10 @@ monitor:
   # endpoints filtering
   # serviceTopology: false
 
+  # -- Enable service load balancing via envoy proxy
+  # This will automatically set enable-envoy-config as well.
+  # envoy: false
```

Comment:
Helm docs unfortunately seems unable to parse those commented out values. Could you set it to a sensible default such that it is documented?

Reply:
good point :100:, I will also keep an eye on below PR

https://github.com/cilium/cilium/pull/21195

Reply:
update the helm values with default value :100: